src/errorHandling/httpError.ts

```diff
@@ -0,0 +1,51 @@
+import ErrorObjectInterface from './errorObjectInterface' // eslint-disable-line no-unused-vars
+
+class HttpError extends Error implements ErrorObjectInterface {
+
+  status:         number;
+  title:          string;
+  message:        string;
+  userMessage:    string;
+
+  constructor(status: number) {
+    super()
+
+    this.status        = 0
```

Comment:
The same comment about the spaces after the `:` applies here. Just 0 or 1 space.